apps/evm/src/clients/api/queries/useGetPools/getPools/getUserTokenBalances/index.ts

```diff
@@ -73,17 +75,39 @@ export const getUserTokenBalances = async ({
       getTokenBalances({
         accountAddress,
         tokens: underlyingTokens,
-        provider,
+        publicClient,
       }),
-      poolLensContract.callStatic.vTokenBalancesAll(isolatedPoolsVTokenAddresses, accountAddress),
-      venusLensContract
-        ? venusLensContract.callStatic.vTokenBalancesAll(legacyPoolVTokenAddresses, accountAddress)
+      publicClient.simulateContract({
+        abi: poolLensAbi,
+        address: poolLensContractAddress as Address,
+        functionName: 'vTokenBalancesAll',
+        args: [isolatedPoolsVTokenAddresses as Address[], accountAddress as Address],
+      }),
+      venusLensContractAddress
+        ? publicClient.simulateContract({
+            abi: venusLensAbi,
+            address: venusLensContractAddress as Address,
+            functionName: 'vTokenBalancesAll',
+            args: [legacyPoolVTokenAddresses as Address[], accountAddress as Address],
```

Comment:
`accountAddress` was updated to the `Address` type, so it looks like it doesn't need the type override